tests/acceptance/async_test_base.py

```diff
@@ -201,12 +233,14 @@ def start(self, adapter_class, ioloop_factory):
         """
         raise NotImplementedError
 
-    def select_default_test(self):
+    @run_test_in_thread_with_timeout
+    def test_with_select_default(self):
```

Comment:
"test" in the beginning of method name allows `py.test` and builtin unittest runners to recognize test methods without customization. Also, it's common practice that functions should be verbs.